api_app/views.py

```diff
@@ -668,3 +684,41 @@ def update(self, request: Request, *args, **kwargs):
             request._full_data = request.data.copy()
         request.data["owner"] = request.user.id
         return super().update(request, *args, **kwargs)
+
+
+@api_view(["DELETE", "POST"])
```

Comment:
in these new views, could you also add the @add_docs?
I would also love to see a logger.info message at the start of each method. Logging is something we always forget until the moment when we need logs and we don't have them and then we cry :P